src/cli/common.rs

```diff
@@ -522,3 +522,18 @@ pub fn report_error(e: &Error) {
         }
     }
 }
+
+pub fn non_fatal_error(error: crate::errors::Error, question: &str, default: bool, no_prompt: bool) -> Result<()> {
```

Comment:
Perhaps "ignorable_error()" might be a better name?

Reply:
Good idea.